spec/Section 3 -- Type System.md

```diff
@@ -1838,78 +1607,50 @@ Expected Type | Internal Value   | Coerced Result
 
 ## Directives
 
-DirectiveDefinition : Description? directive @ Name ArgumentsDefinition? `repeatable`? on DirectiveLocations
+DirectiveDefinition : Description? directive @ Name ArgumentsDefinition? on DirectiveLocations Directives[Const]?
```

Comment:
One thing to consider is that this position would prevent future deprecation of one of the positions:

```
directive @foo on
| QUERY
| MUTATION @deprecated(reason: "Doesn't work well with side effects")
| SUBSCRIPTION
```

(I don't think this is particularly important, but it's worth noting.)

Reply:
You mean because the syntax would become ambiguous, right? I think I'm okay with this (famous last words 😅 )